js/views/chatHeadsVw.js

```diff
@@ -27,7 +27,6 @@ module.exports = baseVw.extend({
     this.$headContainer = $('<div />');
 
     this.showPerScroll = 12;
-    this.nextChatToShow = 0;
```

Comment:
When a new chat convo is initiated via the socket, a new chat head is added to the collection. Manually keeping track of the next chat to show doesn't account for that. So, I made the next chat to show relative to the amount of chat heads  that have been rendered already.